fbw-a32nx/src/behavior/src/A32NX_Interior_ACP.xml

```diff
@@ -289,14 +289,23 @@
             <NORMALIZED_TIME_1>0.1</NORMALIZED_TIME_1>
             <WWISE_EVENT_2>fcubutton</WWISE_EVENT_2>
             <NORMALIZED_TIME_2>0.5</NORMALIZED_TIME_2>
-            <SEQ1_EMISSIVE_CODE>(L:A32NX_ACP#ID#_VOICE_BUTTON_DOWN)</SEQ1_EMISSIVE_CODE>
+            <SEQ1_EMISSIVE_CODE type="rnp" return="number">
+                let dim = 0.0;
+                if (L:A32NX_OVHD_INTLT_ANN, number) == 2 {
+                    dim = 0.1;
+                } else {
+                    dim = 1.0;
+                }
```

Comment:
Please don't create random non-standard emissive code around. We already have templates, or at worst the potentiometer so that the ACP doesn't end up with the wrong brightness next time we adjust lighting or something.